src/pages/index.tsx

```diff
@@ -46,7 +48,9 @@ function renderCardList(tasks: task[], isEditable: boolean) {
       content={item}
       key={item.id}
       shouldEdit={isEditable}
-      onContentChange={async (id: string, newDetails: any) => updateCardContent(id, newDetails)}
+      onContentChange={async (id: string, newDetails: any) => {
+        updateCardContent(id, newDetails, isEditable);
```

Comment:
What is the significance of `{}` here?

Reply:
NIT: Could we think of conditionally calling the function instead of passing it as argument and then checking the condition?

Reply:
What is your opinion on `isEditable && updateCardContent(params)`, would this be better than passing it as params and checking the condition?